Fz10 nn 4

Owczza opened the pull request:
Mam nadzieję, że dobrze to robię. No to tego, jest navbar. Mam nadzieję, że będzie się podobać, jeśli coś jest nie tak to naprawię oczywiście :)

Owczza committed:
Navbar
Added navigation to index.html
Added nav.css
Modified main.css to include margin: 0;, padding: 0; and box-sizing: border-box;

Owczza committed:
Changed background, adjusted mobile responsiveness, added logo.

Owczza committed:
Full responsive navbar
Changed responsiveness to add tablets, fixed minor bugs with the logo and navbar bckgrnd color

Owczza committed:
Switched parameters
Switched parameters to rely on pixels instead of viewport w & height

bbakiewicz commented:
Owca, brakuje bottom-bordera. W kolorze #B66D49. Jego wysokość proponuję na ca 15px. Poza tym cud, miód i orzeszki :)

Owczza committed:
Added bottom border

MagJaw commented on css/main.css:
```diff
@@ -0,0 +1,17 @@
+}
+
+html, body{
+  margin: 0;
```
uwzględnij aktualną treść main.css zapisaną na developie

Owczza committed:
Merge branch 'develop' into FZ10NN-4

Owczza committed:
Merge branch 'develop' into FZ10NN-4

MagJaw commented on index.html:
```diff
@@ -6,15 +6,37 @@
+        <section class="menu__container"> 
+            <!--List container-->
+            <ul class="navList">
+                <li class="navList__element"><a href="#hero">Hero</a></li>
```
Proponuję, aby tekst "Hero" widoczny na stronie, zastąpić "Home".Osoba postronna nie będzie wiedzieć o co chodzi z tym hero

MagJaw commented on index.html:
```diff
@@ -6,15 +6,37 @@
+            <!--List container-->
+            <ul class="navList">
+                <li class="navList__element"><a href="#hero">Hero</a></li>
+                <li class="navList__element"><a href="#function">Funkcje</a></li>
```
Może zamiast "Funkcje" wpisać "Oferta"

MagJaw commented on index.html:
```diff
@@ -6,15 +6,37 @@
+            <ul class="navList">
+                <li class="navList__element"><a href="#hero">Hero</a></li>
+                <li class="navList__element"><a href="#function">Funkcje</a></li>
+                <li class="navList__element"><a href="#contact">Premiera</a></li>
```
Może zamiast "Premiera" wpisać "Formularz"

MagJaw commented on index.html:
```diff
@@ -6,15 +6,37 @@
+        <!--Container For navList for left alignment.-->
+        <section class="menu__container"> 
+            <!--List container-->
+            <ul class="navList">
```
Napisy są słabo widoczne w wersji na laptopa. Może je pogrubić?

MagJaw commented on css/nav.css:
```diff
@@ -0,0 +1,105 @@
+  width: 100%;
+  transition: .5s all ease-in-out;
+  z-index: 1;
+  opacity: 0;
```
zastanawiam się czy opacity jest potrzebne skoro menu__container i tak jest niewidoczne bo mamy top:-120px; do rozważenia

Owczza replied:
Opacity wywaliłam, wygląda dużo lepiej bez tego. Dzięki.

MagJaw commented on css/nav.css:
```diff
@@ -0,0 +1,105 @@
+  left: 0;
+  width: 100%;
+  transition: .5s all ease-in-out;
+  z-index: 1;
```
z-index może do ".menu__switch:checked ~ .menu__container"

Owczza replied:
Zostawiam, jak jest.

MagJaw commented on css/nav.css:
```diff
@@ -0,0 +1,105 @@
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  height: 100%;
```
do rozważenia czy width i height są tu potrzebne

Owczza replied:
Na tę chwilę tak. W przeciwnym razie content się rozjeżdża.

MagJaw commented on css/nav.css:
```diff
@@ -0,0 +1,105 @@
+  .navList__element{  
+    width: 25%;
+    margin: 0;
+    display: flex;
```
do końca nie rozumiem dlaczego flex jest i w navList i w navList element; powinno wystarczyć tylko w navList, ale wówczas nie wiedzieć czemu nie działa justify-content;;
; kombinowałam,ale nie znalazłam lepszego rozw.;zostawmy jak jest

MagJaw commented on css/nav.css:
```diff
@@ -0,0 +1,105 @@
+@media (min-width: 768px){
+  .menu{
+    height: 65px;
+    border-bottom: 15px;
```
border bottom nie miał iść przypadkiem do menu__label?
tutaj nie ma żadnego efektu

Owczza replied:
Zmieniłam, dzięki.

MagJaw reviewed (approved):
generalnie jest bardzo dobrze, stąd approve; przy czym kilka uwag miałam - prośba o ich rozważenie;

bbakiewicz reviewed (approved):
Approve. Jedyne co zwróciło moją uwagę to:
1. pasek grubości 1px pod dolnym boarderem przy rozdzielczości mniejszej niż 768px
2. przyleganie menu do prawej krawędzi okna przy rozdzielczości między 767px a 810px (czepiam się)

Owczza committed:
Touchups
1. Removed opacity from nav.css
2. Added scroll-behavior
3. Moved bottom border width from .menu to .menu__label in nav.css (@media)
4. Changed link display names in navbar
5. Changed font-weight to 600 for link display names (@media)
